lang/tr.po

```diff
@@ -0,0 +1,39 @@
+msgid ""
+msgstr ""
+"Language: tr\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+
+msgid "msg_help_usage"
+msgstr "Kullanım: vrms-rpm [seçenekler]"
+msgid "msg_help_option_ascii"
+msgstr "    Ücretsiz paketler bulunursa veya ücretsiz paketler 10%% den \n"
+	   "	fazlaysa , rms ASCII-art tablosunu göster."
+msgid "msg_help_option_explain"
+msgstr "    Paketleri listelerken, lisansları kanıtlamak için \n"
+       "    ücretsiz / ücretli sınıflandırma göster."
+msgid "msg_help_option_help"
+msgstr "    Yardımı göster ve kapat."
+msgid "msg_help_option_list"
+msgstr "    Ücretli ve ücretsiz lisanları göstermek dışında,\n"
+       "    ismine göre yazdır. Varsayılan 'ücretli'."
+msgid "msg_help_option_version"
+msgstr "    Versiyon numarasını göster ve kapat"
+msgid "msg_list_noargument"
+msgstr "%s: --list seçeneği delil gerektiriyor"
+msgid "msg_list_badargument"
+msgstr "%s: --list seçeneği için delil 'hiç', 'ücretsiz', 'ücretli' or 'hepsi' olmalıdır"
+msgid "msg_unknown_option"
+msgstr "%s: bilinmeyen seçenek '%s'"
+msgid "msg_total_free"
+msgstr "%d ücretsiz paketler"
+msgid "msg_total_nonfree"
+msgstr "%d ücretli paketler"
+msgid "msg_rms_happy_ascii"
+msgstr "Yalnızca bedava paketler - rms gurur duyuyor!                           rms mutlu."
```

Comment:
This particular string should not exceed 79 characters. Please remove some spaces.